py3-attrs.yaml

```diff
@@ -21,21 +21,35 @@ environment:
       - py3-hatchling
       - py3-pip
       - py3-setuptools
+      - py3-wheel
       - python3
       - wolfi-base
 
 pipeline:
-  - uses: fetch
+  - uses: git-checkout
     with:
-      expected-sha256: 935dc3b529c262f6cf76e50877d35a4bd3c1de194fd41f47a2b7ae8f19971f30
-      uri: https://files.pythonhosted.org/packages/source/a/attrs/attrs-${{package.version}}.tar.gz
+      repository: https://github.com/python-attrs/attrs.git
+      tag: ${{package.version}}
+      expected-commit: 9e443b18527dc96b194e92805fa751cbf8434ba9
+
+  - name: Fetch History (required by pip install)
+    runs: |
+      git fetch --unshallow
 
   - name: Python Install
-    runs: pip3 install . --root=${{targets.destdir}}
+    runs: export SOURCE_DATE_EPOCH=$(date +%s) && pip3 install . --root=${{targets.destdir}}
```

Comment:
```suggestion
    runs: export SOURCE_DATE_EPOCH=315532800 && pip3 install . --root=${{targets.destdir}}
```

Reply:
was talking w/ @xnox who suggested that initially but we pivoted to trying to set it to the commit time: https://chainguard-dev.slack.com/archives/C063AT3FVFA/p1717712979583829, falling back to the minimum allowed time like you suggested.